vip-jetpack/connection-pilot/class-jetpack-connection-pilot.php

```diff
@@ -220,8 +226,8 @@ protected function send_alert( $message = '', $wp_error = null, $last_heartbeat
 			$message .= sprintf( ' Jetpack connection error: [%s] %s', $wp_error->get_error_code(), $wp_error->get_error_message() );
 		}
 
-		if ( ( defined( 'WPCOM_SANDBOXED' ) && WPCOM_SANDBOXED ) || 
-			( ! defined( 'ALERT_SERVICE_ADDRESS' ) ) || 
+		if ( ( defined( 'WPCOM_SANDBOXED' ) && WPCOM_SANDBOXED ) ||
```

Comment:
Let's simplify this whole block:
1) we don't really care about any of these constants because on a sandbox the call to Alerts service will fail regardless.
2)Instead of using `error_log` let's use log2logstash (which also writes to error log)

Reply:
Should be good now.